extensions/calendly/package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "calendly",
+  "title": "Calendly",
```

Should we name this a little bit more precise? It's the title of the store entry.

```suggestion
  "title": "Calendly Share Meeting Links",
```